### ISSUE TITLE
Any idea now to make ctr + r work from a terminal in vscode?

### ISSUE THREAD

**Issue by AlexisColes** [kelleyma49/PSFzf#26]
Thanks for this module, very cool.

Although would be even greater if I could get it to work in vs code though, any ideas?

**Comment by kelleyma49** [comment]
Thanks! 

Regarding it not working: I believe it doesn't work in the Integrated Console because PSReadline doesn't seem to work in the IC. If you create a new PowerShell window, PSReadline and PSFzf work.

I'll continue to investigate!

**Comment by kelleyma49** [comment]
@AlexisColes, it looks as though this will be fixed in a future version of the PowerShell vscode extension, but I don't know the ETA.

Here are the relevant tickets:
https://github.com/PowerShell/vscode-powershell/issues/535
https://github.com/PowerShell/vscode-powershell/issues/1793

The workaround is to split a window from the Integrate Console. CTRL+R will work in this new window:
![PSFzfVSCodeTest](https://user-images.githubusercontent.com/2152684/64563669-c0d49d00-d304-11e9-91d4-3c67b9375542.gif)

**Comment by TylerLeonhardt** [comment]
Yes this will be fixed in #1793 so I'm going to mark this as a duplicate.

This is implemented already in the [PowerShell Preview](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell-Preview) extension. Can you install that and give it a try?

**Comment by mattco18** [comment]
Using the PowerShell Preview extension, the chord doesn't work for me.  I also have an issue with continuous completion using PSFzf.  It exits the search and replaces the path with just the directory separator.

**Comment by mattcargile** [comment]
The `fzf` functionality is still broken in the PowerShell Extension terminal. When you `ctrl` + `r` and then `esc` or `enter`, then the whole terminal hangs and doesn't return control to the user. One must restart the whole terminal to regain control.
